components/button/__tests__/index.test.js

```diff
@@ -150,8 +153,11 @@ describe('Button', () => {
       }
     }
     const wrapper = mount(<DefaultButton />);
+    expect(wrapper.render().hasClass('ant-btn-loading')).toBe(false);
     wrapper.simulate('click');
-    expect(wrapper.hasClass('ant-btn-loading')).toBe(false);
+    expect(wrapper.render().hasClass('ant-btn-loading')).toBe(true);
```

Comment:
It should be `false` because we have a delayed loading. I think our loading prop of object type works correctly.